File: angular-front-end/src/app/create-schedule/create-schedule-request.ts
export class CreateScheduleRequest {
  constructor(
    public name: string,
    public duration: string,
    public startDate: string,
    public endDate: string,
    public startTime: string,
    public endTime: string
  ) {  }

  isEmpty(): boolean {
    return !this.name || !this.duration || !this.startDate || !this.endDate || !this.startTime || !this.endTime;
  }
}

File: Algol/src/main/java/edu/wpi/cs/algol/lambda/createschedule/CreateScheduleRequest.java
package edu.wpi.cs.algol.lambda.createschedule;
public class CreateScheduleRequest {
	String name;
	String startDate;
	String endDate;
	String startTime;
	String endTime;
	String duration;
	

	public CreateScheduleRequest(String name, String startDate, String endDate, String startTime, String endTime,
			String duration) {

		this.name = name;
		this.startDate = startDate;
		this.endDate = endDate;
		this.startTime = startTime;
		this.endTime = endTime;
		this.duration = duration;
		
	}


	@Override
	public String toString() {
		return "CreateScheduleRequest [name=" + name + ", startDate=" + startDate + ", endDate=" + endDate
				+ ", startTime=" + startTime + ", endTime=" + endTime + ", duration=" + duration + "]";
	}
	
}
